templates/global/filter_form.html

```diff
@@ -4,10 +4,10 @@ <h3 class="text-center text-black pt-1">House Filter</h3>
     <form
             method="post"
             id="HouseFilter"
-            data-cities-url="{% url 'house:ajax_load_cities' %}"
             novalidate
     >
         <table>
+            {{ form.media.js }}
```

Comment:
What is the media.js file?

Reply:
these are external scripts that the library added on the form so it would be reactive to the changes of the select elements
it is documented here :https://django-smart-selects.readthedocs.io/en/latest/usage.html#usage-in-templates